### PR TITLE
Implements Shared BreadCrumb

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#10234
*Description of changes:*
- Implements Shared Breadcrumb
- Breadcrumb resources
- Removes obsolete code
- Version updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/index.ts [hunk 1]
```diff
@@ -17,4 +17,5 @@ export * from './roleTypes'
 export * from './routeNames'
 
 // external enums
-export { CorpTypeCd, CorpClass, SummaryActions } from '@bcrs-shared-components/enums'
+export { SummaryActions } from '@bcrs-shared-components/enums'
+export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
```
Comment (severinbeauvais) [inline src/enums/index.ts:21]: Why did you change this?

Comment (cameron-freshworks) [reply src/enums/index.ts:21]: When we updated to `"@bcrs-shared-components/enums": "1.0.14"` it wasn't recognizing the `CorpTypeCd` & `CorpClass`, it needed to be explicit. 
I'm not sure what changed, i hadn't been working with the enums, i just updated this at your request 'while i was in there'

Comment (severinbeauvais) [reply src/enums/index.ts:21]: Thanks for the update.

That's weird, since the shared index.ts definitely exports those. Must be another one of those cases where the system doesn't recognize those to be the same (for which we have a ticket).

**Review by severinbeauvais — APPROVED** [review]
LGTM

File: src/App.vue [hunk 2]
```diff
@@ -248,6 +252,28 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
...
+        text: this.getCurrentBusinessName || 'Numbered Benefit Company',
+        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getBusinessId}`
+      },
+      {
+        text: this.entityTitle,
+        to: { name: this.isCorrectionFiling ? RouteNames.CORRECTION : RouteNames.ALTERATION }
+      }
+    ]
+
+    if (this.isRoleStaff) {
```
Comment (severinbeauvais) [inline src/App.vue:268]: Here, too, please add a comment explaining what this code is doing / why it's needed.

Comment (cameron-freshworks) [reply src/App.vue:269]: Added in latest commit

Comment (cameron-freshworks) [reply src/App.vue:270]: Added!
